mysql-test/suite/galera/r/galera_defaults.result

```diff
@@ -24,7 +25,6 @@ WSREP_CERTIFICATION_RULES	strict
 WSREP_CERTIFY_NONPK	ON
 WSREP_CLUSTER_NAME	my_wsrep_cluster
 WSREP_DBUG_OPTION	
-WSREP_DEBUG	NONE
```

Comment:
Why this change?

Reply:
Probably it isn't required, I'll revert this too. At first I thought this param depends on build type, not on CLI parameter.